tests/tfengine/integration_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration_test
```

Comment:
Nit: maybe rename to "tfengine_integration_test" in case we add others later.

Reply:
This is in the tfengine dir, there is a separate dir for policygen, unless you mean to consolidate them to one place?

Reply:
Right, it's already scoped to tfengine.